src/components/Input/_input.scss

```diff
@@ -30,9 +32,14 @@ $lg: #{$px}-input--lg;
     border: 1px solid $keyline-gray;
     border-radius: 0.1875rem;
     font-size: 0.8125rem;
+    font-variation-settings: 'wght' 600;
     margin-bottom: 0.25rem;
     padding: $padding-xsm;
```

Comment:
seems like the bottom padding to the Error is padding-micro in the Figma?

Reply:
sure, I can add in

this would stay the same though, since it's not the error?

Reply:
also this padding value is for the inner padding of the input

Reply:
<img width="640" alt="Screenshot 2024-10-30 at 11 00 20 AM" src="https://github.com/user-attachments/assets/bb2d4f99-b2a5-4c9d-984e-12de8733ace4">

🤔

Reply:
maybe some of that space is coming from the line height of the input